jgrapht-core/src/test/java/org/jgrapht/FastTestSuite.java

```diff
@@ -28,6 +28,7 @@
  */
 @RunWith(ParallelSuite.class)
 @Categories.ExcludeCategory(SlowTests.class)
+@Categories.ExcludeCategory(OptionalTests.class)
```

Comment:
This does not look right! There should be only on `@Categories.ExcludeCategory` with both annotations.

Reply:
I know, I corrected this in the last commit.